app/components/listing/feedback/FeedbackSteps.jsx

```diff
@@ -52,17 +52,15 @@ export const FeedbackTags = ({ tagOptions, onSelectTag }) => (
   </div>
 );
 
-export const Review = ({ reviewValue, onReviewChange, isReviewRequired }) => (
+export const Review = ({ reviewValue, onReviewChange }) => (
   <div className={styles.feedbackReview}>
     <div className={styles.stepsPrompt}>
       Please provide your feedback below:
     </div>
     <textarea
       className={styles.feedbackTextarea}
       type="text"
-      placeholder={`Type your feedback here ${
-        !isReviewRequired ? '(optional)' : ''
-      }`}
+      placeholder="Type your feedback here(optional)"
```

Comment:
It'll look tidier and if you put a space between the word "here" and the parenthetical "(optional)":

```suggestion
      placeholder="Type your feedback here (optional)"
```